test/README.md

```diff
@@ -65,6 +65,64 @@ Note:
 - `AUTO_UPDATE` doesn't work with tests that have multiple `TEST_OUTPUT` segments
 - `AUTO_UPDATE` can be set as an environment variable or as Makefile-like argument assignment
 
+### Running the Unit Tests
+
+The unit tests will automatically run when all tests are run using `./run.d` or
+`make`. To only run the unit tests the `./run.d unit_tests` command can be used.
+For a more finer grain control over the unit tests the `./run.d -u` command can
+be used:
+
+To run all unit tests:
+
+```sh
+./run.d -u
+```
+
+To only run the unit tests in one or more specific files:
+
+```sh
+./run.d -u -- unit/deinitialization.d
```

Comment:
I don't think we need the `--`. It's an anti-pattern that `dub` introduced, there aren't many precedents in other tools for this.

Reply:
I've seen this style in in several tools, e.g. git - `git log -p master -- posix.mak`. It's mostly useful when the CLI has options (named parameters), non-file arguments and file arguments.

Reply:
It's common practice among unix tools that invoke a subcommand to use `--` to separate the arguments for the subcommand from the main command. In this case there will be a conflict with the `-f` flag. `run.d` uses `-f` to mean "force" and `unit_test_runner.d` to mean "filter". I can work around that if you insist.